packages/e2e-tests/specs/editor/various/font-size-picker.test.js

```diff
@@ -98,7 +98,7 @@ describe( 'Font Size Picker', () => {
 		expect( content ).toMatchSnapshot();
 	} );
 
-	it( 'should reset a named font size using input field', async () => {
+	it.skip( 'should reset a named font size using input field', async () => {
```

Comment:
What's the status of this test. Should we enable it, update or remove it completely?

Reply:
Oh, hmm! I didn't know about this. Let me try re-enabling